### ISSUE TITLE
Preview window in search mode

### ISSUE THREAD

**Issue by jrekier** [ihsanturk/neuron.vim#41]
I don't understand how to reproduce the behaviour from the picture below. 
![image](https://user-images.githubusercontent.com/9407736/89715073-768bee00-d9a3-11ea-8291-de557b6ceb82.png)

I can search my zettels while editing with no problem (see below). But how can I get a preview of the highlighted zettel as in the picture above ? 

![image](https://user-images.githubusercontent.com/9407736/89715111-d2567700-d9a3-11ea-8b0e-e96344112ba7.png)

Thanks for your help !

**Comment by ihsanturk** [comment]
Yes that is right. It is an old image. It used to show the preview window, but some updates (using title and id cache without using fzf's with_preview option) brought us to this situation that the plugin searches without preview window. Sorry for that. I will try to bring back the preview window in the feature. I can't focus on this project right now but I am sure I will. If anybody willing to send a pull request at this moment then I would appreciate that. Thanks!

**Comment by fiatjaf** [comment]
Just a shameless self-promotion: I've done this in https://github.com/fiatjaf/neuron.vim/commit/2580487b2ba05e6ca17c8958764e539fcabc2ed3.
